Server/Controllers/MatchController.cs

```diff
@@ -197,13 +206,20 @@ List<string> GetSmartRandomHeroes(bool onMap)
 
                         return player;
                     })
-                    .ToList()
+                    .ToList(),
+                LeaderBoard = topPlayers
+                    .Union(responses.Select(x => new LeaderBoardPlayer()
+                    {
+                        Rating = x.Rating,
+                        SteamId = ulong.Parse(x.SteamId)
+                    }))
+                    .OrderByDescending(lp => lp.Rating)
```

Comment:
Leaderboards should include only top of the players, if we'd want to show/highlight local player/team/whole match we already have enough data to do that on the client